paparazzi/src/main/java/app/cash/paparazzi/accessibility/RenderSettings.kt

```diff
@@ -41,20 +38,24 @@ internal object RenderSettings {
   private val colorMap = mutableMapOf<Int, Color>()
 
   fun getColor(view: View): Color {
-    if (view.id == View.NO_ID) {
-      view.id = generateViewId()
-    }
-    return getColor(view.id)
+    val hashCode = "${view::class.simpleName}(${view.iterableTextForAccessibility})".hashCode()
```

Comment:
what does `iterableTextForAccessibility` do here?

Reply:
ah, reread your description.  so two different TextViews would be colored by different accessible text?  But if they're blank or similar, the tiebreaker is moot.  Is there a better tiebreaker?  Or maybe punt for a future pr?

Reply:
You are correct though, if more than one of the same views in the test has the same content, then they will be rendered with the same overlay color. But, i'm not sure it's worth fighting duplicates there since we're already cutting the max int-range down to 8 possible colors.

I guess the idea is to derive a generally-unique color that won't change if it's hierarchy changes within the layout.